Resources/doc/reference/usage.rst

```diff
@@ -113,6 +113,15 @@ message is receive or created.
         </services>
     </container>
 
+.. code-block:: yaml
```

Comment:
Could you please specify both formats as a list?

Something like this: https://sonata-project.org/bundles/admin/master/doc/reference/getting_started.html#step-3-create-an-admin-service

Related doc code: https://raw.githubusercontent.com/sonata-project/SonataAdminBundle/master/Resources/doc/reference/getting_started.rst